### PR TITLE
[FIX] Fixed css class names and comments

### PR DESCRIPTION
**Pull request by Biki-das**
I found a few css  class names and comments which had odd naming, this invalid class names may have resulted in styles not applied . this Pr fixes those and a fixes few comment typos

### CODE REVIEW

**Comment by Biki-das** [comment]
@debdutdeb could you review?

**Comment by Biki-das** [comment]
@murtaza98 can you review?

File: app/theme/client/imports/general/animations.css [hunk 1]
```diff
@@ -104,7 +104,7 @@
 	}
 }
 
-@keyframes spinh {
+@keyframes spin {
```
Comment (debdutdeb) [inline app/theme/client/imports/general/animations.css:107]: Don't think this is a typo, h for horizontal (rotation across Y axis)

**Review by debdutdeb — COMMENTED** [review]
Thanks for the contribution! rest lgtm :) I'll ask someone who can confirm.

File: app/theme/client/imports/general/animations.css (hunk 1 above)
Comment (Biki-das) [reply app/theme/client/imports/general/animations.css:107]: ok sure will revert it

**Pull request closed without merging**